jest.config.js

```diff
@@ -236,13 +234,15 @@ module.exports = {
   // A map from regular expressions to paths to transformers
 
   transform: {
-    '^.+\\.(ts|tsx|js)$': 'ts-jest',
+    '^.+\\.(ts|tsx)$': 'ts-jest',
```

Comment:
WHy js needed to remove?

Reply:
I reverted my changes that was done to make imported `filterTypes.js` to work. Here is how it was before:
![image](https://user-images.githubusercontent.com/36186912/130784754-ba9f5293-e5d3-42b7-9cb7-32a2f836829b.png)
https://github.com/iTwin/iTwinUI-react/pull/266/files#diff-1e058ca1442e46581b13571fb8d261f9e1f5657e26c96634d4c1072f0f0347f1L232

Reply:
ok..
As long as you tested and it works, I would suggest releasing this as patch.